src/openrct2-ui/windows/NewCampaign.cpp

```diff
@@ -173,7 +172,7 @@ class NewCampaignWindow final : public Window
         campaign.no_weeks = 2;
 
         // Currently selected ride
-        campaign.RideId = SELECTED_RIDE_UNDEFINED;
+        campaign.rideId = RideId::GetNull();
```

Comment:
Why did you change this TitleCase to camelCase? We are moving towards the former.

Reply:
Hence the note about using `_t` suffixes for types or `m_` prefixes for data members. There's a whole lot, but changing them all is something clang-tidy could probably do.

Reply:
Adding _t to every type makes the code look a bit strange over time, this means every custom type should have that. We probably have to go over our own code style again as it has problematic aspects using PascalCase for public members. So where does this leave us with this PR?

Reply:
For this PR, it's probably best to stick with our current style with PascalCase, and then use `::RideId` where you need to access the variable instead of the type.

Reply:
I don't think this will work but I'll give it a try on the CI.

Reply:
All good now.